lib/msal-angular/src/msal.interceptor.spec.ts

```diff
@@ -906,9 +908,9 @@ describe("MsalInterceptor", () => {
       sampleAccountInfo,
     ]);
 
-    httpClient.get("http://site.com/relative/me").subscribe();
+    httpClient.get("http://localhost:9876/relative/me").subscribe();
```

Comment:
Note: This unit test needed to be fixed as it was providing a correct match where the host name and port were different in the case of a protected resource that was a relative URL.